samples/SocketsSample/Hubs/HubTChat.cs

```diff
@@ -38,14 +48,14 @@ public Task SendToGroup(string groupName, string message)
 
         public async Task LeaveGroup(string groupName)
         {
-            await Groups.RemoveAsync(Context.ConnectionId, groupName);
```

Comment:
Why did you change this order?

Reply:
So that you see the "left group" message when you leave.